docs/building.md

```diff
@@ -1,6 +1,6 @@
 # Building Slang From Source
 
-## Get the Source Code
+## Get the Source Code 
```

Comment:
Oh no! Whitespace added to end of line!

(No reason to change, but also no reason to leave it in)

Reply:
It's only there because I appveyor failed spuriously, so I did this to kick it to rebuild. I guess there might be a better way from the control panel...